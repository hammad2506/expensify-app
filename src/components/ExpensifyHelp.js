import React from 'react';

const ExpensifyHelp = () => (
    <div>
        <h3>Welcome to the help page</h3>
    </div>
);

export default ExpensifyHelp;