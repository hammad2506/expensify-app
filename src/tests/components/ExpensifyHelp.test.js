import React from 'react';
import { shallow } from 'enzyme';
import ExpensifyHelp from '../../components/ExpensifyHelp';

test("Testing Expensify Help Page Component", () => {
    const wrapper =  shallow(<ExpensifyHelp />);
    expect(wrapper).toMatchSnapshot();
});