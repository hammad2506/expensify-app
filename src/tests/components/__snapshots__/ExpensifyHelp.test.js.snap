// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Expensify Help Page Component 1`] = `
<div>
  <h3>
    Welcome to the help page
  </h3>
</div>
`;
